core/widgetdiv.ts

```diff
@@ -69,11 +70,19 @@ export function createDom() {
   if (document.querySelector('.' + containerClassName)) {
     containerDiv = document.querySelector('.' + containerClassName);
   } else {
-    containerDiv = document.createElement('div') as HTMLDivElement;
+    containerDiv = document.createElement('div');
     containerDiv.className = containerClassName;
   }
+  if (!containerDiv) return;
```

Comment:
Is this case actually possible or just TS not knowing that it isn't? The code above seems to guarantee the div will exist, so perhaps the code could be rearranged to ensure that TS is aware of the value being present in order to obviate this check.